src/Magnum/Audio/Renderer.h

```diff
@@ -188,6 +240,21 @@ class Renderer {
 /** @debugoperatorclassenum{Magnum::Audio::Renderer,Magnum::Audio::Renderer::Error} */
 Debug MAGNUM_AUDIO_EXPORT operator<<(Debug debug, Renderer::Error value);
 
+/** @debugoperatorclassenum{Magnum::Audio::Renderer,Magnum::Audio::Renderer::DistanceModel} */
+Debug MAGNUM_AUDIO_EXPORT operator<<(Debug debug, Renderer::DistanceModel value);
+
+inline Vector3 Renderer::listenerPosition() {
+    Vector3 position;
+    alGetListenerfv(AL_POSITION, position.data());
+    return position;
+}
+
+inline std::array<Vector3, 2> Renderer::listenerOrientation() {
+    std::array<Vector3, 2> data = {Vector3{}, Vector3{}};
```

Comment:
Initializing the array is not needed, it's done implicitly and will be overwritten by the getter below anyway.